### PR TITLE
feat:homepage and assets folder

### PR DESCRIPTION
**Pull request by ultronjs**
Homepage
-Home Banner
-Featured Category
-Pick your accessories
-Games from top brands
Assets Folder

### CODE REVIEW

**Review by Namrata-J — APPROVED** [review]

File: app.css [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+.container_featured_category,.container_games_from_top_brands{
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: space-around;
+    gap: 0.5rem;
+}
+.container_pick_your_accessories{
+    display: grid;
+    grid-template-columns: repeat(auto-fit,minmax(222px,1fr));
```
Comment (dev-enforced) [inline app.css]: in minmax you can use rem or em too instead of px.

File: app.css [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    min-height: 14vh;
+    background-color: black;
+    color: white;
+    padding: 1rem 1rem;
+    display: flex;
+    justify-content: space-between;
+}
+ul{
+    padding-inline-start: 0.5rem;
+ }
```
Comment (dev-enforced) [inline app.css]: Can shift it to reset.css file where you can pass the list-style-type:none and this property of padding-inline-start

**Review by dev-enforced — APPROVED** [review]
The code looks good to me!! One component that you can take care of the footer is:
image
image

Give some gap between the links and spread it horizontally instead of vertically

File: app.css (hunk 1 above)
Comment (ultronjs) [reply app.css]: done

File: app.css (hunk 2 above)
Comment (ultronjs) [reply app.css]: done

**Comment by ultronjs** [comment]
Merging after resolving comments

**Pull request merged**